ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/logProcessor/leRobot/LeRobotDatasetTools.java

```diff
@@ -1,6 +1,6 @@
 package us.ihmc.avatar.logProcessor.leRobot;
 
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Apparently I accidentally used an old version of apache here transitively provided through rosjava.